__tests__/app.test.js

```diff
@@ -176,3 +176,77 @@ describe("6.GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+describe.only("7.POST/api/articles/:article_id/comments", () => {
+  test("responds with statusCode 201 and responds with the posted comment", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "hjhgfhjf",
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toMatchObject({
+          comment_id: 19,
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+          author: "butter_bridge",
+          body: "hjhgfhjf",
+        });
+      });
+  });
+  test("responds with statusCode 400 if the user sends no body", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "",
```

Comment:
Please write another test to make sure that unnecessary properties are ignored, for example when a new comment has an extra property: `{username: "butter_bridge, body: "A brand new comment.", isBestComment: "true"}`.

Your code seems fine selecting only the right properties, but this test should be included.